lib/utils/addEmptyLineBefore.js

```diff
@@ -1,11 +1,25 @@
 /* @flow */
 'use strict';
 
-// Add an empty line before a node. Mutates the node.
+const _ = require('lodash');
+
+/** @typedef {import('postcss').ChildNode} ChildNode */
+
+/**
+ * Add an empty line before a node. Mutates the node.
+ *
+ * @param {ChildNode} node
+ * @param {'\n' | '\r\n'} newline
+ * @returns {ChildNode}
+ */
 function addEmptyLineBefore(
 	node /*: postcss$node*/,
 	newline /*: '\n' | '\r\n'*/,
 ) /*: postcss$node*/ {
+	if (_.isUndefined(node.raws.before)) {
```

Comment:
Since there is a tend to reduce using `lodash` maybe it's a case to use just
```js
node.raws.before === undefined
```
Under the hood that's all it does.
And also in the `lib/utils/addEmptyLineAfter.js`

Reply:
You're right! `isUndefined` looks strange now 😂